File: WebApp/Models/MI/MI_UNITCODE.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class MI_UNITCODE : JCLib.Mvc.BaseModel
    {
        public string UNIT_CODE { get; set; }
        public string UI_NAME { get; set; }     // 自訂 UNIT_CODE + UI_CHANAME
        public string UI_CHANAME { get; set; }
        public string UI_ENGNAME { get; set; }
        public string UI_SNAME { get; set; }
        public string CREATE_DATE { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_DATE { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
    }
}

File: WebAppVen/Scripts/app/model/MI_UNITCODE.js
Ext.define('WEBAPP.model.MI_UNITCODE', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'UNIT_CODE', type: 'string' },
        { name: 'UI_CHANAME', type: 'string' },
        { name: 'UI_ENGNAME', type: 'string' },
        { name: 'UI_SNAME', type: 'string' },
        { name: 'CREATE_DATE', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_DATE', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },

        // =================== 湘倫 ======================
        { name: 'UI_NAME', type: 'string' },     // 自訂 UNIT_CODE + UI_CHANAME
        // ===============================================
    ]
});